waspc/data/Generator/templates/react-app/src/actions/core.js

```diff
@@ -5,10 +5,10 @@ import {
 } from '../operations/resources'
 
 export function createAction(actionRoute, entitiesUsed) {
-  async function internalAction(args, optimisticallyUpdatedCacheKeys) {
-    registerActionInProgress(optimisticallyUpdatedCacheKeys)
+  async function internalAction(args, optimisticUpdateTuples) {
```

Comment:
TODO: come up with a better name for these guys (if only we had types... :upside_down_face:)

Reply:
Came up with a better name and added a JSDoc documenting their type (albeit in a different file). I couldn't get JSDoc to work across files. There are some suggestions on how to make it work online (we'd have to edit/add a `jsconfig.json` file) , but they're probably not worth it. Hopefully we'll have TypeScript soon enough.